app/lib/devise_recognisable/guard.rb

```diff
@@ -70,6 +76,17 @@ def compare_ip_addresses(session_address)
     distance < Devise.max_ip_distance
   end
 
+  # Method to check if the user agent strings are similar. Uses the Levenshtein
+  # distance to calculate the minumum number of changes required to make an
+  # exact match. Takes a session user agent string and returns a bool.
+  def compare_user_agents(session_user_agent)
+    return true if session_user_agent == @request.user_agent
+
+    distance = DamerauLevenshtein.distance(@request.user_agent, session_user_agent, 0)
```

Comment:
Add a comment that the third arg forces this gem to use traditional Levenshtein.